server/helpers/tag_helpers.py

```diff
@@ -0,0 +1,50 @@
+import calendar
+from typing import Dict, List
+
+from server.models import Tag
+
+
+def build_categorised_tags(tags: List[Tag]) -> Dict:
+
+    categorised_tags = {}
```

Comment:
is this a set or dictionary?
Make it clear by using `dict()`

Reply:
In my experience this is a fairly common convention, it's a set when there is something between the brackets (set comprehension). However! Explicit is better than implicit so why not be safe rather than sorry, so agreed